functions/go/project-service-list/README.md

```diff
@@ -0,0 +1,117 @@
+# project-services-list
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `project-services-list` function generates [project service](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service)
```

Comment:
I think it will help to describe the overall motivation to help user understand why and when to use this function.

Reply:
added a line to describe why/when to use this